easybuild/easyconfigs/i/IMOD/IMOD-4.7.15_RHEL6-64_CUDA6.0.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'Binary'
+
+name = 'IMOD'
+version = '4.7.15'
+versionsuffix = '_RHEL6-64_CUDA6.0'
+
+homepage = 'http://bio3d.colorado.edu/imod/'
+description = """IMOD is a set of image processing, modeling and display
+programs used for tomographic reconstruction and for 3D reconstruction of EM
+serial sections and optical sections. The package contains tools for assembling
+and aligning data within multiple types and sizes of image stacks, viewing 3-D
+data from any orientation, and modeling and display of the image files. IMOD
+was developed primarily by David Mastronarde, Rick Gaudette, Sue Held, Jim
+Kremer, Quanren Xiong, and John Heumann at the University of Colorado."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+toolchainopts = {'openmp': True}
+dependencies = [('CUDA', '6.0.37')]
+
+source_urls = ['http://bio3d.colorado.edu/imod/AMD64-RHEL5/']
+sources = ['%(namelower)s_%(version)s%(versionsuffix)s.csh']
+
+install_cmd = "tcsh %(namelower)s_%(version)s%(versionsuffix)s.csh"
+
+# -dir: Choose location of installation directory
+# -skip: do not attempt to deploy resource files in /etc
+# -yes: do not prompt for confirmation
+installopts = "-dir %(installdir)s -skip -yes"
+
+# The assumption by the install script is that installdir will be something
+# like /usr/local. So it prefixes the install location with a lot of specific
+# directories that are surplus to requirements.
+postinstallcmds = [	"mv %(installdir)s/%(name)s/* %(installdir)s/.",
+			"rm -r %(installdir)s/%(name)s %(installdir)s/%(namelower)s_%(version)s"
+			]
```

Comment:
How sure are you that this doesn't break anything?
The installation prefix doesn't get hardcoded anywhere in the installed files?

also, please fix indent:

```python
postinstallcmds = [
    "mv %(installdir)s/%(name)s/* %(installdir)s/.",
    "rm -r %(installdir)s/%(name)s %(installdir)s/%(namelower)s_%(version)s",
]
```